tests/cases/standalone/common/types/vector/vector.result

```diff
@@ -31,6 +31,180 @@ SELECT * FROM t;
 | 1970-01-01 00:00:00.003000 | "[7,8,9]" |
 +----------------------------+-----------+
 
+SELECT cos_distance(v, '[0.0, 0.0, 0.0]') FROM t;
+
++-------------------------------------------+
+| cos_distance(t.v,Utf8("[0.0, 0.0, 0.0]")) |
++-------------------------------------------+
+| 1.0                                       |
+| 1.0                                       |
+| 1.0                                       |
++-------------------------------------------+
+
+SELECT *, cos_distance(v, '[0.0, 0.0, 0.0]') as d FROM t ORDER BY d;
+
++-------------------------+--------------------------+-----+
+| ts                      | v                        | d   |
++-------------------------+--------------------------+-----+
+| 1970-01-01T00:00:00.001 | 0000803f0000004000004040 | 1.0 |
+| 1970-01-01T00:00:00.002 | 000080400000a0400000c040 | 1.0 |
+| 1970-01-01T00:00:00.003 | 0000e0400000004100001041 | 1.0 |
++-------------------------+--------------------------+-----+
+
+SELECT cos_distance('[1.0, 2.0, 3.0]', v) FROM t;
+
++-------------------------------------------+
+| cos_distance(Utf8("[1.0, 2.0, 3.0]"),t.v) |
++-------------------------------------------+
+| 5.960464477539063e-8                      |
+| 0.025368154048919678                      |
+| 0.04058808088302612                       |
++-------------------------------------------+
+
+SELECT *, cos_distance('[1.0, 2.0, 3.0]', v) as d FROM t ORDER BY d;
+
++-------------------------+--------------------------+----------------------+
+| ts                      | v                        | d                    |
++-------------------------+--------------------------+----------------------+
+| 1970-01-01T00:00:00.001 | 0000803f0000004000004040 | 5.960464477539063e-8 |
+| 1970-01-01T00:00:00.002 | 000080400000a0400000c040 | 0.025368154048919678 |
+| 1970-01-01T00:00:00.003 | 0000e0400000004100001041 | 0.04058808088302612  |
++-------------------------+--------------------------+----------------------+
+
+SELECT cos_distance(v, v) FROM t;
```

Comment:
We may need to round the results to avoid potential sqlness inconsistence on different platforms. see https://github.com/GreptimeTeam/greptimedb/pull/4956